testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -12,7 +12,7 @@ use clarity::vm::costs::ExecutionCost;
 use clarity::vm::types::serialization::SerializationError;
 use clarity::vm::types::PrincipalData;
 use clarity::vm::{ClarityName, ClarityVersion, ContractName, Value, MAX_CALL_STACK_DEPTH};
-use rand::{Rng, RngCore};
+use rand::Rng;
```

Comment:
I think this can be removed now.

Reply:
WHY BRICE!?

Reply:
This one is actually still needed. Its used in make_signed_microblock